blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxReader.java

```diff
@@ -298,19 +298,6 @@ public String getLastStringValue(Locator locator) {
         return points;
     }
 
-    public static String getUnitString(Locator locator) {
```

Comment:
@tilogaat Why are we removing this method? I thought the users will still have an option of getting the unit from `MetadataCache`, if they do not have ES?